NitroxClient/Communication/MultiplayerSession/MultiplayerSessionManager.cs

```diff
@@ -78,7 +78,6 @@ public void RequestSessionReservation(PlayerSettings playerSettings, Authenticat
         public void ProcessReservationResponsePacket(MultiplayerSessionReservation reservation)
         {
             Reservation = reservation;
-            CurrentState.NegotiateReservation(this);
```

Comment:
Should we be removing this?

Reply:
Sorry @killzoms but SR is correct. This is the state that that session manager should be in upon receiving a confirmed reservation:

```suggestion
            CurrentState.NegotiateReservation(this);
```
https://github.com/SubnauticaNitrox/Nitrox/blob/master/NitroxClient/Communication/MultiplayerSession/ConnectionState/AwaitingSessionReservation.cs#L8